app/customer.py

```diff
@@ -0,0 +1,38 @@
+import dataclasses
+import math
+
+from app.car import Car
+from app.shops import Shop
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int
+    car: Car
+
+    def __repr__(self) -> None:
+        return self.name
+
+    def get_distance_to_shop(self, shop_location: list) -> float:
+        return math.sqrt((shop_location[0] - self.location[0]) ** 2
+                         + (shop_location[1] - self.location[1]) ** 2)
+
+    def calculate_road_cost(self, shop: Shop, fuel_price: float) -> float:
+        return round(
+            self.get_distance_to_shop(shop.location)
+            * self.car.fuel_consumption / 100 * fuel_price * 2,
+            2)

```

Comment:
fix the breaking line here, u can move closer qoute to the next line, or move 2 to the previous line